prometheus/prometheus.yml

```
@@ -0,0 +1,12 @@
+global:
+  scrape_interval: 15s
+  evaluation_interval: 15s
+
+scrape_configs:
+  - job_name: 'gosearch'
+    metrics_path: '/metrics'
+    scheme: http
+    static_configs:
+      - targets:
+          - 'gosearch.dk:8081'
+      
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)